src/iOS/toga_iOS/widgets/base.py

```diff
@@ -78,6 +78,17 @@ def add_child(self, child):
         if self.container:
             child.viewport = self.root.viewport
             child.container = self.container
+        # The highest level box doesn't have a container - it is one
+        elif getattr(self, "viewport", None):
```

Comment:
Is the getattr() check necessary here? The two cases are "top level container" and "child container"; this elif implies the existence of a third class (and I'm not sure what that would be... something without a container *or* a viewport?)

Reply:
I'll double check, the getattr may have been from an earlier version of my poking around!